python-avd/pyavd/tests/test_pyavd.py

```diff
@@ -0,0 +1,137 @@
+import glob
+from collections import ChainMap
+from os import path, system
+
+import pytest
+from pyavd import eos_cli_config_gen, eos_designs_facts, eos_designs_structured_configs
+from yaml import safe_dump as yaml_dump
+
+from .utils import create_common_vars, read_vars, write_result
+
+
+@pytest.mark.parametrize(
+    "common_varfiles, fact_file, device_varfiles",
+    [
+        (
+            [
+                "pyavd/vendor/tests/eos_designs/main.yml",
+                "pyavd/vendor/tests/eos_designs/default-connected-endpoints-keys.yml",
```

Comment:
please rebase, since the latest version use fewer default files.

Reply:
Will be part of PR #27